### PR TITLE
Feat/version api

### PR DESCRIPTION
**Pull request by vviers**
This PR: 
- allows the API to log the commit and dataset id it was built on
- these infos are also available directly from the `/version` route of the API

Right now, this work by building the docker with the following command :
`docker build --build-arg=COMMIT=$(git rev-parse --short HEAD) --tag cgus-api:latest .`

But I am not sure if this command will work in ansible.

What do you think ?

### CODE REVIEW

**Comment by vviers** [comment]
TODO: rajouter date `last_updated` sur `/version`

File: app/main.py [hunk 1]
```diff
@@ -42,6 +54,21 @@ async def index(request: Request):
...
+    """
+    Return the current dataset version used by the API
+    The list of dataset releases is available at
+     https://github.com/ambanum/OpenTermsArchive-versions/releases
+    Also return the commit SHA on which the API was built
+    """
+    return {
+        "dataset_version": os.getenv("MOST_RECENT_DATASET", "unknown"),
+        "api_version": os.getenv("COMMIT_SHA", "unknown"),
+    }
```
Comment (martinratinaud) [inline app/main.py:107]: Can you please add lastUpdated

**Review by martinratinaud — DISMISSED** [review]

File: app/main.py (hunk 1 above)
Comment (vviers) [reply app/main.py:107]: This is what the `/version` route returns now:
Screenshot 2021-03-16 at 17 22 44

**Comment by vviers** [comment]
Still have to solve the Ansible issue.

I should be able to work it out with a mix of `build.args` https://docs.ansible.com/ansible/latest/collections/community/docker/docker_image_module.html#ansible-collections-community-docker-docker-image-module 
and 
https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#registering-variables

**Review by martinratinaud — APPROVED** [review]
Seems ok to me

Let me know how it goes when you push to prepro

**Pull request merged**